public_src/components/toolbar/toolbar.component.html

```diff
@@ -67,11 +67,33 @@
     <i class="fa fa-search" aria-hidden="true"></i>
     &nbsp;{{ 'Show' | translate }}
   </button>
-  <button type="button" class="btn btn-default col-md-6 visible-md visible-lg" container="body"
-          tooltip="{{ 'Open JOSM and show selected element' | translate }}"
-          (click)="getLoadAndZoomUrl()"
-          [disabled]="isDisabled()">
-    <i class="fa fa-download" aria-hidden="true"></i>
-    &nbsp;{{ 'Open in JOSM' | translate }}
-  </button>
+
+  <div id="dropdown" class="btn-group col-md-6" dropdown>
+
+     <button dropdownToggle type="button" class="btn btn-default btn-block dropdown-toggle" container="body"
+             tooltip="{{ 'Open in JOSM/ID' | translate }}"
+             [disabled]="isDisabled()">
+            &nbsp;{{ 'Open in ..' | translate }} 
```

Comment:
Please use "Open in ..." (with three dots) and add/remove texts from language files https://github.com/dkocich/osm-pt-ngx-leaflet/blob/develop/public_src/assets/i18n/en.json (keep it sorted alphabetically and keep string as a value when you do not know the translation - we will translate it later in POEditor.com)